lesson01/assignments/lesson01.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Lesson 01 Assignment</title>
+  <!--jQuery CDN -->
+  <script
+      src="https://code.jquery.com/jquery-3.3.1.js"
+      integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60="
+      crossorigin="anonymous">
+  </script>
+<!--Local Files -->
+<script src="js.js"></script>
+</head>
+<body>
+  <h1>Using ES6+ Methods</h1>
+  <!-- First input box -->
+  <div>
+    <h3>ES6+ Reduce Method</h3>
+    <!-- Start first input box -->
+    <input type="text" class="input" id="firstInput">
+    <!-- Submit for Input -->
+    <input class="submit" type="submit" id="firstSubmit">
+    <!-- Result markup with hook for answer -->
+    <p>Result after adding the inputted value to the sum of the values in my array: <span class="resultOne"></span></p>
+  </div>
+  <!-- End first input box -->
+<br>
+  <!-- Second input box -->
+  <div>
+    <h3>ES6+ Map Method</h3>
+    <!-- Start second input box -->
+    <input type="text" class="input" id="secondInput">
+    <!-- Submit for Input -->
+    <input class="submit" type="submit" id="secondSubmit">
+    <!-- Result markup with hook for answer -->
+    <p>Result after multiplying the numbers in my array by the inputted value: <span class="resultTwo"></span></p>
+  </div>
+  <!-- End second input box -->
+<br>
+  <!-- Third input box -->
+  <div>
+    <h3>ES6+ Filter Method</h3>
+    <!-- Start third input box -->
+    <input type="text" class="input" id="thirdInput">
+    <!-- Submit for Input -->
+    <input class="submit" type="submit" id="thirdSubmit">
+    <!-- Result markup with hook for answer -->
+    <p>Numbers in my array that are less than the inputted value: <span class="resultThree"></span></p>
+  </div>
+  <!-- End third input box -->
+<br>
+  <!-- Fourth input box -->
+  <div>
+    <h3>ES6+ find method</h3>
+    <!-- Input box -->
+    <input type="text" class="input" id="fourthInput">
+    <!-- Submission -->
+    <input class="submit" type="submit" id="fourthSubmit">
+    <!-- Explanation with result -->
+    <p>The first element in my array that is more than the inputted value: <span class="resultFour"></span></p>
+  </div>
```

Comment:
Consider making names more semantic. Instead of ids "fourthInput" and "fourthSubmit", "findInput" and "findSumbit" would more clearly link them with their functionality, and make future maintenance easier.